frontend/components/combobox/ComboboxTopics.vue

```diff
@@ -29,6 +30,7 @@
         >
           <ComboboxOptions
             class="absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-light-distinct dark:bg-dark-distinct py-1 text-base elem-shadow-lg ring-1 ring-black/5 focus:outline-none sm:text-sm"
+            id="isVisibleElement"
```

Comment:
We just need the `id` to go above class here given the [eslint order](https://github.com/activist-org/activist/blob/ff0f8b0bfcf2f665236c7f3e5480e12f0006a307/frontend/.eslintrc.js#L40), @UnknownSean8 :) Also happy to do this myself!